### DISCUSSION TITLE
Good idea, but unfortunate execution. Don't recommend using this.

### DISCUSSION THREAD

**Discussion by zombri-eats-brainz** [fenneishi/Fooocus-Control#28] [General]
I could write a lengthy essay about everything wrong with this version of fooocus. I came for controlnet, but what I found were numerous useless changes, missing/broken command line arguments, and nonsensical prompt expansion. I wasted a lot of time and huge numbers of generations messing with this. Why? Lets say you run this in realistic mode. Somehow the prompt expansion throws in things like "oil painting", "illustration", "classic anime from 2010 in the style of Hayao Mizyazaki", "digital painting", and other such nonsense. Can't just set a seed and test something. I spent far too many hours messing with prompts trying to fix strange outputs and really none of it mattered. Also absolutely hate how <lora: ______ : x.x> is completely ignored in the prompts, so we're literally stuck with only 5 possible LoRAs loaded. 

I hope the original makers of Fooocus come snag whatever controlnet stuff actually works from this.

**Comment by rionix88** [comment]
seems like they abandoned this project. no update anymore
